data.sql

```diff
@@ -7,3 +7,44 @@ VALUES ('Agumon', '2020-02-03', 0, true, 10.23),
 ('Gabumon', '2018-11-15', 2, true, 8),
 ('Pikachu', '2021-01-07', 1, false, 15.04),
 ('Devimon', '2017-05-12', 5, true, 11);
+
+-- 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES 
+('Charmander', '2020-02-08', 0, false, -11),
+('Plantmon', '2022-11-15', 2, true, -5.7),
+('Squirtle', '1993-04-02', 3, false, -12.13),
+('Angemon', '2005-07-12', 1, true, -45),
+('Boarmon', '2005-06-07', 7, true, 20.4),
+('Blossom', '1998-10-13', 3, true, 17);
+
+BEGIN;
+UPDATE animals 
+SET species = 'unspecified';
+ROLLBACK;
```

Comment:
* I suggest you properly state these insertions `Inside a transaction update the animal's table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction.`  
* You only started the transaction, made the change (` UPDATE animals SET species = 'unspecified';`) but you didn't verify that change was made and undone for example:  ` SELECT species from animals;` 
  ` ROLLBACK;`
Then
  ` SELECT species from animals; `
`